build/ndm-daemonset/entrypoint.sh

```diff
@@ -9,10 +9,10 @@ else
   # set ulimit to unlimited and create a core directory for creating coredump
   echo "[entrypoint.sh] enabling core dump."
   ulimit -c unlimited
-  echo "[entrypoint.sh] creating $SPARSE_FILE_DIR/core if not exists."
-  mkdir -p "$SPARSE_FILE_DIR/core"
-  echo "[entrypoint.sh] changing directory to $SPARSE_FILE_DIR/core"
-  cd "$SPARSE_FILE_DIR/core" || exit
+  echo "[entrypoint.sh] creating /var/openebs/core if not exists."
+  mkdir -p "/var/openebs/core"
```

Comment:
Is it good to do something like
```
cd /var/openebs || exit
mkdir -p core
cd core
```
So that even if users given wrong value in mount paths we will not allow.

Reply:
Done.